search_in_timelines_and_encode_non_studio.py

```diff
@@ -3,6 +3,9 @@
 #Modify the variable in nextline ("v05") to a string you want to search for. Then Copy and paste all text to console.
 
 searchintimelinenamesfor = "_v05"
+renderFormat = "mp4"
+renderCodec = "H264"
+targetDir = "/Users/Luis/Desktop/videoResults"
```

Comment:
Path is absolute. Better set to relative:
from pathlib import Path
home = str(Path.home())